docs/agents/sematext-agent/app-agent/index.md

```diff
@@ -0,0 +1,60 @@
+title: App Agent
+description: App Agent is used for collecting application metrics from JMX, REST APIs, and from databases that support JDBC as well as transaction traces collection and on demand JVM profiling.
+
+The App Agent is open-source, written in Java (but not limited to
+monitoring Java!), with a number of built-in integrations available
+from the
+[sematext-agent-integrations](https://github.com/sematext/sematext-agent-integrations)
+repo. You can add your own integrations for monitoring additional
+infrastructure and services.  Pull requests welcome!
+
+## Capabilities
+
+The agent communicate with Sematext via HTTPS.  It buffers data on
+disk in case it cannot immediately ship it, ensuring data is never
+lost.
+
+### Application monitoring
+
+Lightweight and pluggable, this agent comes with a number of out of
+the box integrations. It has built-in support for collecting metrics
+from JMX, REST APIs, and from databases that support JDBC.
+
+### Tracing
+
+The Sematext agent can also instrument JVM-based apps using Java bytecode
+instrumentation to collect [transaction traces](../tracing).
+
+### Profiling
+
+On demand [JVM profiling](on-demand-profiling) is built-in and help
+you find bottlenecks in your code.
+
+## App Agent Run modes
+
+### Embedded
+
+The [Embedded](spm-monitor-javaagent) mode can be used only for
+monitoring Java-based applications since it runs as a Java Agent
```

Comment:
We should probably say "JVM-based", not Java-based.  You could monitor e.g. Kotlin or even Scala.